### PR TITLE
 chore(docs): updated readme

### PR DESCRIPTION
**Pull request by BasMasking**
Fixes #

Changes proposed in this pull request:
- Added integration examples to examples readme
- Added integration examples to main readme

@MaskingTechnology/jitar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +139,12 @@ For splitting (segmenting) applications configuration files are used. For each p
 
 There are more practical [examples](examples/README.md) in the repository. Practical guides are provided in the [publications](#publications) section.
 
+## Integrations
+
+Jitar provides integration examples for the following frameworks:
```
Comment (petermasking) [inline README.md:144]: frontend frameworks

File: README.md [hunk 2]
```diff
@@ -138,6 +139,12 @@ For splitting (segmenting) applications configuration files are used. For each p
...
+- [react-jitar](examples/4-integrations/1-react/README.md)
```
Comment (petermasking) [inline README.md:145]: [React]

File: README.md [hunk 3]
```diff
@@ -138,6 +139,12 @@ For splitting (segmenting) applications configuration files are used. For each p
...
+- [vue-jitar](examples/4-integrations/2-vue/README.md)
```
Comment (petermasking) [inline README.md:146]: [Vue]

File: README.md [hunk 4]
```diff
@@ -138,6 +139,12 @@ For splitting (segmenting) applications configuration files are used. For each p
...
+
```
Comment (petermasking) [inline README.md:147]: We are working on integrations with more frontend and meta frameworks, so this list will grow over time.

**Review by petermasking — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (BasMasking) [reply README.md:147]: Processed

File: README.md (hunk 3 above)
Comment (BasMasking) [reply README.md]: Processed

File: README.md (hunk 2 above)
Comment (BasMasking) [reply README.md]: Processed

File: README.md (hunk 1 above)
Comment (BasMasking) [reply README.md]: Processed

**Review by petermasking — APPROVED** [review]

**Pull request merged**